CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -1422,7 +1423,7 @@
       </trans-unit>
       <trans-unit id="With a subscription, you get:" xml:space="preserve">
         <source>With a subscription, you get:</source>
-        <target state="translated">З підпискою, ви отримуєте:</target>
+        <target state="translated">За передплатою, ви отримаєте:</target>
```

Comment:
Don't think that a comma is needed in Ukrainian translation

Reply:
I asked an expert (my ex 😅) and she says comma is not needed - I trust her much more than myself in this regard